### ISSUE TITLE
Add UI for changing default upload settings per project

### ISSUE THREAD

**Issue by edjubuh** [purduesigbots/pros-cli#53]
Currently, if you want to change default upload options, you have to edit the project.pros file manually and know the correct keys to edit. It would be nice to have a checkbox in the upload project prompt to "use these settings next time."

**Comment by HotelCalifornia** [comment]
bumping this to high priority because I think this should be done before we release interactive UI
